_posts/2016-09-27-clean.markdown

```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Clean
+date: 2016-09-27 00:00:00
+homepage: https://github.com/knaman2609/clean  
+download: https://github.com/knaman2609/clean/archive/master.zip
+demo: http://knaman2609.github.io/clean/
+author: knaman2609
+thumbnail: clean.png
+license: MIT License
+---
```

Comment:
Hello! If You don't have a license file in Your repository, then please add http://opensource.org/licenses/MIT as `license_link`. Thanks!